docs/userguide/configuring.rst

```diff
@@ -133,21 +150,22 @@ Here are a series of question to help formulate a suitable configuration:
           are on a **native Slurm** system like :ref:`configuring_nersc_cori`
 
 
-4. Where will you run the main parsl process vs the tasks?
-
-+---------------------+--------------------------+------------------------------------+
-| Workflow location   | Execution target         | Suitable channel                   |
-+=====================+==========================+====================================+
-| Laptop/Workstation  | Laptop/Workstation       | `LocalChannel`                     |
-+---------------------+--------------------------+------------------------------------+
-| Laptop/Workstation  | Cloud Resources          | None                               |
-+---------------------+--------------------------+------------------------------------+
-| Laptop/Workstation  | Clusters with no 2FA     | `SSHChannel`                       |
-+---------------------+--------------------------+------------------------------------+
-| Laptop/Workstation  | Clusters with 2FA        | `SSHInteractiveLoginChannel`       |
-+---------------------+--------------------------+------------------------------------+
-| Login node          | Cluster/Supercomputer    | `LocalChannel`                     |
-+---------------------+--------------------------+------------------------------------+
+4. Where will you run the main Parsl program, given that you already have determined where the apps will run?
+(This is needed to determine how to communicate between the Parsl program and the apps.)
```

Comment:
It's not clear to the reader what happens over a channel.  Is this where inputs and outputs are passed?  containerized apps? files?  I'm not sure where we explain this, but it would be useful to probably say a little more here.

Reply:
I'm not sure if that level of detail is necessary/useful to the user while deciding what config pieces to use.

Reply:
@danielskatz agreed, but can you split this off into either a PR which adds the proposed content or an issue to remind us to add these details? We don't want to hold up merging your nice improvements above by waiting on more work to be done.